governance/votes/Trial Executive vote - July 12, 2023.md

```diff
@@ -0,0 +1,191 @@
+
+**[Dai Savings Rate (DSR)](https://manual.makerdao.com/parameter-index/core/param-dai-savings-rate) Decrease**
+- Decrease the DSR by 0.30% from 3.49% to 3.19%
+
+**[Liquidation Ratio](https://manual.makerdao.com/parameter-index/vault-risk/param-liquidation-ratio) Changes**
+- Reduce WSTETH-A Liquidation Ratio by 10% from 160% to 150%
+- Reduce WSTETH-B Liquidation Ratio by 10% from 185% to 175%
+- Reduce RETH-A Liquidation Ratio by 20% from 170% to 150%
+
+**[Stability Fee](https://manual.makerdao.com/parameter-index/vault-risk/param-stability-fee) Reductions**
+- Reduce the ETH-A Stability Fee (SF) by 0.30% from 3.74% to 3.44%
+- Reduce the ETH-B Stability Fee (SF) by 0.30% from 4.24% to 3.94%
+- Reduce the ETH-C Stability Fee (SF) by 0.30% from 3.49% to 3.19%
+- Reduce the WSTETH-A Stability Fee (SF) by 0.30% from 3.74% to 3.44%
+- Reduce the WSTETH-B Stability Fee (SF) by 0.30% from 3.49% to 3.19%
+- Reduce the RETH-A Stability Fee (SF) by 0.30% from 3.74% to 3.44%
+- Reduce the WBTC-A Stability Fee (SF) by 0.11% from 5.80% to 5.69%
+- Reduce the WBTC-B Stability Fee (SF) by 0.11% from 6.30% to 6.19%
+- Reduce the WBTC-C Stability Fee (SF) by 0.11% from 5.55% to 5.44%
+
+#### CRVV1ETHSTETH-A 1st Stage Offboarding
+
+As per [MIP104 ](https://mips.makerdao.com/mips/details/MIP104#0-the-stability-scope) the debt ceiling (``line``) for ``CRVV1ETHSTETH-A`` will be set to ``0``. 
+
+This change will be happening alongside the other [Stability Scope Parameter changes](https://forum.makerdao.com/t/stability-scope-parameter-changes-3/21238/6).
+
+
+### Deploy Multiswap Conduit for RWA015-A (Andromeda)
+
+Following the results of a recent [technical assessment from ChainSecurity](https://github.com/makerdao/rwa-toolkit/blob/8d30ed2cb657641253d45b57c894613e26b4ae1b/audits/ChainSecurity_MakerDAO_RWA_Toolkit_audit.pdf), the following changes will be made to [BlackTower Andromeda RWA015](https://github.com/makerdao/rwa-toolkit) if this proposal passes:
+
+- A new version of the output conduit will be added to the chainlog as `RWA015_A_OUTPUT_CONDUIT`. This will allow an authorized party to choose which PSM they want to interact with.
+- The operator on the old output conduit will be de-authorized.
+- `RWA015_A_OUTPUT_CONDUIT_LEGACY` will be removed from the chainlog.
+
+More context around [RWA015] Project Andromeda: Technical Assessment can be found [here](http://forum.makerdao.com/t/rwa015-project-andromeda-technical-assessment/20974/11).
+
+### Deploy FlapperUniV2
+
+The Smart Burn Engine (Dss Flappers) allocates excess Dai from the Surplus Buffer and accumulates MKR tokens in the form of Univ2 LP tokens. Surplus Auctions (flaps) will be replaced by the [Dss Flappers](https://github.com/makerdao/dss-flappers/blob/univ2/README.md) contracts ([audit](https://chainsecurity.com/security-audit/maker-flapperuniv2/)).
```

Comment:
Generally I would order detail on 'what is happening' below the statement on 'why this is happening'. Eg, starting with: 'Due to this poll/ratified doc/auth/etc, x will happen if this is approved'

'x involves...